storagemarket/types/dealcheckpoints/checkpoints.go

```diff
@@ -20,7 +20,7 @@ var names = map[Checkpoint]string{
 	Published:           "Published",
 	PublishConfirmed:    "PublishConfirmed",
 	AddedPiece:          "AddedPiece",
-	IndexedAndAnnounced: "IndexedAndAnnounced",
+	IndexedAndAnnounced: "Indexed",
```

Comment:
Let's leave this as IndexedAndAnnounced because clients may be relying on it to have that name